--- rote-0.2.6/Makefile.in~	2004-10-06 12:35:38.000000000 +0000
+++ rote-0.2.6/Makefile.in	2004-10-19 03:54:49.670772712 +0000
@@ -39,26 +39,26 @@
 all: librote.so.$(ROTE_VERSION)
 
 install: all
-	mkdir -p $(includedir)/rote
-	rm -f $(includedir)/rote/*.h
-	cp rote.h $(includedir)/rote
-	mkdir -p $(libdir)
-	cp librote.so.$(ROTE_VERSION) $(libdir)
-	cd $(libdir) && ln -sf librote.so.$(ROTE_VERSION) librote.so
-	cd $(libdir) && ln -sf librote.so.$(ROTE_VERSION) $(ROTE_SONAME)
+	mkdir -p $(DESTDIR)$(includedir)/rote
+	rm -f $(DESTDIR)$(includedir)/rote/*.h
+	cp rote.h $(DESTDIR)$(includedir)/rote
+	mkdir -p $(DESTDIR)$(libdir)
+	cp librote.so.$(ROTE_VERSION) $(DESTDIR)$(libdir)
+	cd $(DESTDIR)$(libdir) && ln -sf librote.so.$(ROTE_VERSION) librote.so
+	cd $(DESTDIR)$(libdir) && ln -sf librote.so.$(ROTE_VERSION) $(ROTE_SONAME)
 	chmod 755 rote-config
-	mkdir -p $(bindir)
-	cp -p rote-config $(bindir)
+	mkdir -p $(DESTDIR)$(bindir)
+	cp -p rote-config $(DESTDIR)$(bindir)
 	@echo "-----------------------------------------------------------"
 	@echo "ROTE - Our Own Terminal Emulation Library v$(ROTE_VERSION)"
 	@echo
-	@echo "Include files installed at: $(includedir)"
-	@echo "Library files installed at: $(libdir)"
-	@echo "rote-config executable    : $(bindir)/rote-config"
+	@echo "Include files installed at: $(DESTDIR)$(includedir)"
+	@echo "Library files installed at: $(DESTDIR)$(libdir)"
+	@echo "rote-config executable    : $(DESTDIR)$(bindir)/rote-config"
 	@echo
 	@echo "To find out what compiler arguments you should use to"
 	@echo "compile programs that use rote, use the rote-config"
-	@echo "program (make sure $(bindir) is in your path)."
+	@echo "program (make sure $(DESTDIR)$(bindir) is in your path)."
 	@echo "-----------------------------------------------------------"
         
 librote.so.$(ROTE_VERSION): $(OBJECTS)
